pkgs/applications/window-managers/sway/default.nix

```diff
@@ -34,7 +27,7 @@ stdenv.mkDerivation rec {
   buildInputs = [
     wayland libxkbcommon pcre json_c dbus libevdev
     pango cairo libinput libcap pam gdk_pixbuf
-    wlroots wayland-protocols
+    wlroots wayland-protocols swaybg makeWrapper
```

Comment:
Unless I'm mistaken the swaybg here is still unnecessary

Reply:
I believe `makeWrapper` should also be a native build input

Reply:
No, because it used in making the wrapper.

Reply:
you pass it in with `"${swaybg}/bin"` though

Reply:
Oh, yeah, you're right.